.gitignore

```diff
@@ -8,6 +8,7 @@
 /tmp/
 /.bundle/
 /tmp/
+/.idea/
```

Comment:
I am a RubyMine user, so I added this directory to the `.gitignore` file.
I don't feel strongly about that, though, so please just let me know if you'd like me to remove it.
🙇